### PR TITLE
Encoded input generation

### PR DESCRIPTION
**Pull request by swarooprm**
- Added a function to generated encoded input files and kept it in src/utils
- Removed BART_Sample Prediction.json
- Updated Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,19 +15,26 @@ predictions/bart_outputs
 The BART predictions, in particular, correspond to a model that was trained on a random subset of tasks and evaluated on the remaining ones. 
 
 ## Evaluation script 
-The script that we used in our evaluation is included in `src/???`. 
+The script that we used in our evaluation is included in `src/evaluation.py`. 
+
+## How to Evaluate
```
Comment (danyaljj) [inline README.md]: Make this a subsection of the evaluation section?  (add a "#")

File: README.md [hunk 2]
```diff
@@ -15,19 +15,26 @@ predictions/bart_outputs
...
+It requires dataset file path and the prediction file path
+E.g.  
+python3 evaluation.py --predictions ../predictions/gpt3_outputs/subtask002_quoref_answer_generation@_Definition_Prompt@0_100.json --dataset ../Dataset_Jsons/subtask002_quoref_answer_generation.json
```
Comment (danyaljj) [inline README.md:25]: Make this a code snippet (inside ```s)

File: README.md [hunk 3]
```diff
@@ -15,19 +15,26 @@ predictions/bart_outputs
...
+
+Filenames in GPT3_Outputs are of the format [taskname]'@'[instruction encoding]'@'[number of examples]'_'[number of instances].json
```
Comment (danyaljj) [inline README.md]: Similarly, add code highlighting to the fille name ("`"s on the sides)

File: README.md [hunk 4]
```diff
@@ -15,19 +15,26 @@ predictions/bart_outputs
...
+## How to Evaluate
+It requires dataset file path and the prediction file path
+E.g.  
+python3 evaluation.py --predictions ../predictions/gpt3_outputs/subtask002_quoref_answer_generation@_Definition_Prompt@0_100.json --dataset ../Dataset_Jsons/subtask002_quoref_answer_generation.json
+
+Filenames in GPT3_Outputs are of the format [taskname]'@'[instruction encoding]'@'[number of examples]'_'[number of instances].json 
 
 ## Encoding the instructions 
-TODO 
+In 'src/utils', encodeinstruction function is provided to generate encoded instruction inputs.
```
Comment (danyaljj) [inline README.md]: Link the file: [`encodeinstructions.py`](link)

File: README.md [hunk 5]
```diff
@@ -15,19 +15,26 @@ predictions/bart_outputs
...
+E.g.
+encodeinstruction('subtask003_mctaco_question_generation_event_duration', instruction_structure =['Definition','Prompt'])
```
Comment (danyaljj) [inline README.md:33]: make it a code snippet

**Comment by danyaljj** [comment]
Added some comments. Also please resolve the conflicts with the master branch.

**Comment by swarooprm** [comment]
I have resolved all comments, please let me know if I need to change anything else.

**Comment by danyaljj** [comment]
LGTM! 
Merging ...

**Pull request merged**